### PR TITLE
MGMT-6046 Align cluster Ready status with telemetry's Ready

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

**Comment by mareklibra** [comment]
@ukalifon This is expected to break tests.
Once merge, we will not have a cluster in the `Ready` state, just in `Draft`. The tests will need to reimplement related conditions.

File: src/config/constants.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const HOST_ROLES: HostRole[] = [
 export const CLUSTER_STATUS_LABELS: { [key in Cluster['status']]: string } = {
   'pending-for-input': 'Draft',
   insufficient: 'Draft',
-  ready: 'Ready',
+  ready: 'Draft',
```
Comment (mareklibra) [inline src/config/constants.ts:61]: Cc @ukalifon

File: src/components/clusters/ClusterStatus.tsx [hunk 2]
```diff
@@ -18,10 +18,12 @@ import { CLUSTER_STATUS_LABELS } from '../../config/constants';
 
 type ClusterStatusProps = {
   status: Cluster['status'];
+  className?: string;
```
Comment (jtomasek) [inline src/components/clusters/ClusterStatus.tsx:21]: I don't see the `className` used anywhere?

Comment (mareklibra) [reply src/components/clusters/ClusterStatus.tsx:21]: Good catch. But it is used from the uhc-portal where a value is actually passed.

**Review by jtomasek — APPROVED** [review]

File: src/components/clusters/ClusterStatus.tsx (hunk 2 above)
Comment (jtomasek) [reply src/components/clusters/ClusterStatus.tsx:21]: I've been greping uhc-portal codebase for `<ClusterStatus` and couldn't see it anywhere?

**Pull request merged**